projects/templates/projects/view_project.html

```diff
@@ -149,6 +149,36 @@ <h4>Current Users</h4>
   {% endfor %}
 </ul>
 
+{% if is_pi %}
+  <h4>Pending Invitations</h4>
+
+  <form class="form-inline" role="form" method="post">
+    {% csrf_token %}
+    {% bootstrap_form invite_form %}
+    <div class="form-group">
+      <button type="submit" class="btn btn-default" name="invite_user">Invite user</button>
+    </div>
+  </form>
+
+  <ul class="list-unstyled">
+    {% for i in invitations %}
+    <li>
+      {{i.email_address}}
+      <form role="form" method="post" style="display:inline">
+        {% csrf_token %}
+        <input type="hidden" name="email_code" value="{{ i.email_code }}">
+        <button type="submit" class="btn btn-xs btn-link" name="resend_invite"><i class="fa fa-envelope text-info"></i><span class="sr-only">Resend invitation</span></button>
+      </form>
+      <form role="form" method="post" style="display:inline">
+        {% csrf_token %}
+        <input type="hidden" name="email_code" value="{{ i.email_code }}">
+        <button type="submit" class="btn btn-xs btn-link" name="del_invite"><i class="fa fa-minus-square text-danger"></i><span class="sr-only">Remove invitation</span></button>
+      </form>
+    </li>
+    {% endfor %}
+  </ul>
```

Comment:
Can you attach a screenshot of what this looks like / where it is relative to other UI? I imagined it would be around the project membership section.

Reply:
![image](https://user-images.githubusercontent.com/9397092/123461904-46474100-d5af-11eb-8877-cfb61fde4aea.png)
Let me know what you think would be best. One thing I just noticed is the input is below the title here, but the "Current Users" title and the input to add new users are opposite of this. We could include more space between the Users and Invites sections.

Reply:
Another thing I notice here is clearing the input after submitting.